security.md

```diff
@@ -37,3 +37,90 @@ We are working to identify and coordinate security efforts across the Jupyter co
+
+# Jupyter Vulnerability Handling Process
+
+This document summarizes and proposes guidelines for handling vulnerabilities reported in Jupyter projects. 
+
+Security issues and vulnerabilities have expectations and processes that are differ from typical open source practices:
+ - Private discussions
+ - Obfuscation
+ - Short timeline
+
+This makes it quite hard to be able to understand, learn, or know what to expect from a security point of view. This document
+will give you a glimpse on what's happening on the inside, and what timeline to expect when you report a security vulnerability. 
+It will also serve as a guideline and task list for Jupyter Contributors and Developers on how to handle security relatied issues.
+
+## Scope
+
+This process applies to *all projects* governed by Jupyter, including JupyterLab, Jupyter Notebook, JupyterHub, and Jupyter Server.
+
+## Reporting Vulnerabilities
+
+If you believe you’ve found a security vulnerability in a Jupyter project, please report it to security@ipython.org. If you prefer 
+to encrypt your security reports, you can use [this PGP public key](https://jupyter.org/assets/ipython_security.asc). Project Jupyter 
+will respond within 3 days to all new reports.
+
+## Coordinated Disclosures
+
+Project Jupyter follows a [coordinated disclosure](https://cheatsheetseries.owasp.org/cheatsheets/Vulnerability_Disclosure_Cheat_Sheet.html#responsible-or-coordinated-disclosure)
+model where the initial report and remediation are handled privately, but the completion description is made public once a patch
+is available. Project Jupyter will disclose known vulnerabilities within 90 days by default, whether a patch is available or not.
+
+## Acknowledgement
+
+Project Jupyter will work to ensure that security researchers, developers, users, or others who identify and report vulnerabilities
+within Project Jupyter software receive acknowledgement for their contribution. 
+
+
+## Vulnerability Triage & Remediation Process
+
+This section describes the process used by Project Jupyter to track, remediate, and disclose reported vulnerabilities. 
+This description is both a reference for the Jupyter Community and a guideline for contributors.
+
+### Roles
+
+This process defines these roles:
+- **Reporter** The individual(s) who report the vulnerability 
+- **Coordinator** A Project Jupyter contributor who facilitates the tracking of the vulnerability through this process
+- **Developer** One or more Project Jupyter developers who work on remediating the vulnerability
+
+For the purpose of this document these roles are distinct, in practice, some of these roles may be handled by the same individual. However, the roles should be covered by a minimum of two separate individuals. For example, a Reporter may also fill the Developer role and create the remediation, in this case the Coordinator should be a separate individual.
+
+### Process
+
+The role responsible for each step is noted at the beginning.
```

Comment:
I like the process perspective, but it may be useful to have another section that describes the responsibilities of the different roles.